resources/css/humhub.wiki.less

```diff
@@ -5,23 +5,23 @@
 
 @info: #21A1B3;
 @warning: #FFC107;
-@text-color-main: #555;
+@text-color-main: var(--text-color-main);
 
-@text-color-black: #000;
-@text-color-darkgrey: #AEAEAE;
-@text-color-grey: #7A7A7A;
+@text-color-black: var(--text-color-highlight);
```

Comment:
I don't agree to use var with different name/meaning, because if value of the `--text-color-highlight` will be changed on some theme then here it will be unexpected to see for example red color where we expect to see black color.

Same for 4 vars below `@text-color-darkgrey`, `@text-color-grey`, `@bg-color-white` and `@border-color-black`.

We could use the core/general variables if they mean almost same as the current vars. I think we cannot apply a color from core var which was developed for another purpose, i.e. border color can be applied for some other border var and etc.

Probably we should think again how the CSS variables can be used here in the Wiki module, because this LESS file was created before we implemented the global CSS variables.

Reply:
Thank you! So I close this PR.